### ISSUE TITLE
Do not rebundle unchanged files

### ISSUE THREAD

**Issue by brettstack** [swimlane-contrib/systemjs-route-bundler#36]
Here's a challenge. Currently it's painful to develop based on bundled builds (which you may want to do for debugging issues that only occur in the bundled version) because the entire app needs to be rebundled for every change. This could take several minutes (or longer; depending on the size of your app). It would be really awesome if the bundler were smart enough to only bundle changes in the tree.

**Comment by amcdnl** [comment]
Eh, that gets really tricky.  Sure you could do this semi-easily with a gulp watch task but the problem is if you add/remove a file, the tree needs to be rebuilt entirely since the change could have cascading effects.

**Comment by brettstack** [comment]
Yeah it's certainly a challenge. I just wanted to post an issue here for the backlog. Maybe someone will want to pick it up.
